cvat/apps/engine/views.py

```diff
@@ -2105,7 +2110,82 @@ def content(self, request, pk):
             # need to update index
             manifest.set_index()
             manifest_files = [os.path.join(manifest_prefix, f) for f in manifest.data]
-            return Response(data=manifest_files, content_type="text/plain")
+            return Response(
+                data=manifest_files,
+                content_type='text/plain',
+                headers={'Deprecation': datetime(2023, 7, 1, tzinfo=timezone.get_current_timezone()).strftime('%a, %d %B %Y %H:%M:%S %Z')}
```

Comment:
I just wanted to emphasize in every possible way that this endpoint will be deprecated. Yes, I assumed this date, because I don't know in advance the date of the next release. But I can't indicate in the `Deprecation` header that this endpoint will be deprecated in the next release due to specification.
So, do you recommend removing it at all?

Reply:
Oh, thank you, for some reason I missed it when I looked at the spec